properties/src/main/java/com/fasterxml/jackson/dataformat/javaprop/JavaPropsSchema.java

```diff
@@ -211,6 +221,23 @@ public JavaPropsSchema withPathSeparator(String v) {
         return s;
     }
 
+    /**
+     * Mutant factory method for constructing a new instance with
+     * specified path separator escape; default being null ('\0') which
+     * should effectively disable escape processing.
+     * Any escape character may be used, backslash ('\\') is the most obvious candidate
```

Comment:
Based on code I think this will only work for escaping path separator character itself (at least initial index check
verifies it is followed by path separator)
That is fine but should probably be explicitly mentioned so that users do not try to use it for escaping other characters. Might be also worth mentioning that JDK properties loader will handle Unicode escapes?
(a link to explanation would be great if one can be found).

Reply:
I've expanded the documentation on both the field and the method, making them both very similar.
I've also put the same content in the README.MD (hopefully I've got the markdown right).